### DISCUSSION TITLE
Config do site da Flask Conf 2023

### DISCUSSION THREAD

**Discussion by biancarosa** [apyb/comunidade#32] [Geral]
Fala pessoal, precisamos configurar o site da Flask Conf 2023 :) 

URL: http://2023.flask.python.org.br.
Redirecionar  http://flask.python.org.br para http://2023.flask.python.org.br.

O site está no gh pages, repo: https://github.com/flask-brasil/flaskconf-site-2023, CNAME ja esta  configurado no github :)

**Comment by rougeth** [comment]
✅ Feito.
```
❯ dig +short 2023.flask.python.org.br CNAME
flask-brasil.github.io.
```

Legal @biancarosa, as dadas já foram confirmadas?

**Reply by biancarosa** [reply]
Tentando me organizar pra fazer nessas datas hehe, mas não temos local ainda.

**Comment by biancarosa** [comment]
oi @rougeth!

consegue confirmar pra mim como funciona esse redirect:

 http://flask.python.org.br/ -> http://2023.flask.python.org.br/.


 http://flask.python.org.br/ tá indo pra  http://2019.flask.python.org.br/

**Reply by rougeth** [reply]
@biancarosa vou precisar de mais tempo pra ver isso, não sei como/onde isso ta configurado

**Reply by biancarosa** [reply]
caso ajude:

```
% dig flask.python.org.br +short                                                                                                         ~
ip-45-33-120-99.cloudezapp.io.
45.33.120.99
% dig 2023.flask.python.org.br +short                                                                                                    ~
flask-brasil.github.io.
185.199.111.153
185.199.108.153
185.199.109.153
185.199.110.153
```

**Reply by rougeth** [reply]
`flask.python.org.br` aponta para esse servidor `ip-45-33-120-99.cloudezapp.io`, nele existe uma cópia (literalmente) do site de 2019, servida por um Nginx. Não temos documentado como isso foi feito, nem como atualizar.

Github [não suporta](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages) múltiplos domínios apontados para o mesmo projeto, eles sugerem configurar um redirecionamento no serviço de DNS (o que usamos hoje em dia não fornece essa opção).

Para agora, poderíamos apontar o `flask.python.org.br` para `flask-brasil.github.io`, mas ficaríamos sem o `2023.` até acharmos outra opção.

O que acha @biancarosa?

**Reply by biancarosa** [reply]
Acho tranquilo! Podemos usar o 2023 como o flask.python.org.br até chegar a hora de lançar o proximo, dai mudamos o atual pra 2023.

**Reply by rougeth** [reply]
Atualizei, acho que agora só mudar o CNAME do seu lado.

**Comment by biancarosa** [comment]
Erro ainda @rougeth :

DNS check unsuccessful
flask.python.org.br is improperly configured
Domain's DNS record could not be retrieved. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidDNSError).
